cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -164,72 +175,122 @@ bool CobTwistController::initialize()
     ros::Time last_update_time_ = time_;
     ros::Duration period_ = time_ - last_update_time_;
 
-    this->limiters_.reset(new LimiterContainer(this->twistControllerParams_, this->chain_));
+    this->twist_controller_params_.keep_direction = true;
+    this->twist_controller_params_.enforce_pos_limits = true;
+    this->twist_controller_params_.enforce_vel_limits = true;
+
+    this->limiters_.reset(new LimiterContainer(this->twist_controller_params_, this->chain_));
     this->limiters_->init();
 
     ROS_INFO("...initialized!");
     return true;
 }
 
-void CobTwistController::reconfigure_callback(cob_twist_controller::TwistControllerConfig &config, uint32_t level)
+void CobTwistController::reinitServiceRegistration()
+{
+    ROS_INFO("Reinit of Service registration!");
+    std::string robo_namespace = nh_.getNamespace();
+    robo_namespace.erase(0, 1); // delete "/"
+    ros::ServiceClient client = nh_.serviceClient<cob_obstacle_distance::Registration>(robo_namespace + "/obstacle_distance/registerPointOfInterest");
+    ROS_INFO_STREAM("Created service client for service " << robo_namespace << "/obstacle_distance/registerPointOfInterest");
```

Comment:
so what is the resolved name of this service?
```/arm_right/obstacle_distance/registerPointOfInterest```?
Isn't then the additional ```robo_namespace``` incorrect? (```robo_namespace``` is equal to ```/arm_right```, right?)